Source/Extensions/Blazorise.Cropper/CropperState.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Blazorise.Infrastructure;
+
+namespace Blazorise.Cropper;
+
+/// <summary>
+/// Provides a shared state and syncronization context between the cropper and cropper viewer.
+/// </summary>
+public class CropperState
```

Comment:
why is this public? The user cannot manipulate the state and also not do anything with it. Make internal? See comment at the CropperViewer aswell

Reply:
It must be public or otherwise, it wouldn't be able to declare and assign to the Cropper and CropperViewer.

Reply:
PS. A also had in mind to add more features to the CropperState in the future. I took this as an inspiration https://github.com/aspnet/AspLabs/blob/main/src/QuickGrid/src/Microsoft.AspNetCore.Components.QuickGrid/Pagination/PaginationState.cs